vit/config_parser.py

```diff
@@ -166,7 +168,7 @@ def get(self, section, key):
         try:
             value = self.config.get(section, key)
             return self.transform(key, value, default)
-        except (configparser.NoSectionError, configparser.NoOptionError):
+        except (configparser.NoSectionError, configparser.NoOptionError, ValueError):
```

Comment:
Why did you add `ValueError` here?

Reply:
Because if you parse int or float you can get ValueError. In that case you need to use default value.